src/components/Onboarding.jsx

```diff
@@ -0,0 +1,42 @@
+import logo from '../assets/logo.png'
+import * as rb from 'react-bootstrap'
+import React, { useState } from 'react'
+
+export default function Onboarding() {
+  const [step, setStep] = useState(0)
+
+  if (step === 0) {
+    return (
+      <div class="mx-auto my-auto">
+        <div class="d-flex justify-content-center">
+          <img src={logo} alt="logo" class="mb-4" width="200rem" />
```

Comment:
You can use the sprite with logo here to ensure it works across the themes:

```suggestion
          <Sprite symbol="logo" width="200" height="200" className="mb-4" />
```